app/cmd/preview.go

```diff
@@ -182,18 +216,83 @@ var previewCmd = &cobra.Command{
 			CLIBenchmark: bench,
 		})
 		if err != nil {
-			cleanUpFiles()
+			removeArtifacts()
 			learn.API.NotifySlack(err)
 			os.Exit(1)
 		}
 	},
 }
 
+// maybeCreateNewTarget will set up and create everything needed for single file uploads if they are needed.
+// Returns a string representing the source name which if not single file tmp dir is needed, will return the
+// original
+func maybeCreateNewTarget(target string, singleFileImagePaths []string) (string, error) {
+	if len(singleFileImagePaths) > 0 {
```

Comment:
can we invert the logic and return `target, nil` early rather than scootch the contents of the function over two spaces?